blog-posts/enterprises-benefit-from-scoped-npm-registries.md

```diff
@@ -51,9 +51,11 @@ Even if the registry is hosted internally, it is good practice to add authentica
 
 `username:password` ⇒ base64 encoded ⇒ `dXNlcm5hbWU6cGFzc3dvcmQ=`
 
-For encoding you can use the command line `echo -n 'my-string' | base64` (Linux) or an IDE (vscode with plugin) or search for an online base64 encoder.
+For encoding you could use the command line `echo -n 'my-string' | base64` (bash) or an IDE (e.g. vscode with plugin). Or alternatively you could even use a REPL (e.g. [JS REPL](https://nodejs.dev/learn/how-to-use-the-nodejs-repl) with the following command in JavaScript `Buffer.from('username:password', 'binary').toString('base64')`).
```

Comment:
There is a "shorthand" for that `Buffer.from(...)` command: `btoa()` (https://developer.mozilla.org/de/docs/Web/API/btoa).

![image](https://user-images.githubusercontent.com/7970458/150305604-d91bebd6-4ab3-4453-a0eb-eac7ec23e255.png)


```suggestion
For encoding you could use the command line `echo -n 'my-string' | base64` (bash) or an IDE (e.g. vscode with plugin). Or alternatively you could even use a REPL (e.g. [JS REPL](https://nodejs.dev/learn/how-to-use-the-nodejs-repl) with the following command in JavaScript `btoa('username:password')`).
```

Reply:
Interesting, didn't know that!

Reply:
@fischeversenker maybe I did something wrong but I tried it with the btoa() command but it fails:
![image](https://user-images.githubusercontent.com/10372711/150510140-843aeb37-ae00-4e4f-8a5b-215559dc56d0.png)

Reply:
Turns out this is not yet available in node 14. As you can see initially I'm in node 16 where it works, then I switch to node 14 (`nvm use 14`) and try again to see it fail:
![image](https://user-images.githubusercontent.com/7970458/150527979-3057910d-35d2-436a-92ea-5b28c5b6cc34.png)

I also had a look at the documentation. It turns out `atob` and `btoa` were added in Node 16 but they are already considered Legacy: https://nodejs.org/dist/latest-v16.x/docs/api/globals.html#btoadata
So ignore this and stick to the `Buffer.from(...)` approach 👌

Reply:
:raised_hands: good to know :+1: